src/styles/index.scss

```
@@ -1,3 +1,144 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&family=Roboto:wght@400;500&display=swap");
+
 body {
   margin: 0;
 }
+
+h2 {
+  margin-top: 40px;
+  margin-bottom: 4px;
+  width: 166px;
+  color: var(--Main-accent, #060b35);
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px; /* 150% */
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+}
+
+.content {
+  margin: 33px 17px 17px;
+
+  &__code {
+    color: var(--Secondary, #616070);
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px; /* 140% */
```

Remove comments everywhere
```suggestion
    line-height: 14px;
```